modules/Bio/EnsEMBL/Compara/Locus.pm

```diff
@@ -449,8 +433,7 @@ sub get_sequence {
         # Coordinates are 0-based
         $seq = $faidx_helper->get_sequence2_no_length($self->dnafrag_id, $self->dnafrag_start-1, $self->dnafrag_end-1);
         die "sequence length doesn't match !" if CORE::length($seq) != ($self->dnafrag_end-$self->dnafrag_start+1);
-        reverse_comp(\$seq) if $self->dnafrag_strand < 0;
-
+        reverse_comp(\$seq) if (defined $self->dnafrag_strand) && ($self->dnafrag_strand < 0);
```

Comment:
On what occasions is `dnafrag_strand` not defined?

Reply:
If I recall correctly, during my testings there were cases when a warning was printed because of that, so I added that to avoid unnecessary warnings.

Reply:
I feel like `dnafrag_strand` should always be defined, in the DBs it is `NOT NULL`, could there be another issue somewhere leading to these warnings?